scripts/auto-rebase/rebase.py

```diff
@@ -138,66 +168,82 @@ def try_get_rebase_branch_ref_from_remote(remote, branch_name):
-        return True
+
+    logging.info(f"Remote branch is older - it needs updating. "
+                    f"Remote branch is on top of {base_branch_name}'s commit: '{commit_str(remote_merge_base[0])}'. "
+                    f"Local branch is on top of {base_branch_name}'s commit '{commit_str(local_merge_base[0])}'")
+    return True
 
 
 def try_get_pr(gh_repo, org, base_branch, branch_name):
+    """
+    Try to get a pull request for a branch on a GitHub repository.
+    Returns
+    - The pull request if it exists and is open, otherwise None.
+    - If more than one pull request is found, then the first one will be used.
+    """
     prs = gh_repo.get_pulls(base=base_branch, head=f"{org}:{branch_name}", state="all")
 
     if prs.totalCount == 0:
         logging.info(f"PR for branch {branch_name} does not exist yet on {gh_repo.full_name}")
         return None
 
-    pr = None
+    pull_req = None
     if prs.totalCount > 1:
-        pr = prs[0]
-        logging.warning(f"Found more than one PR for branch {branch_name} on {gh_repo.full_name} - this is unexpected, continuing with first one of: {[(x.state, x.html_url) for x in prs]}")
+        pull_req = prs[0]
+        logging.warning(f"Found more than one PR for branch {branch_name} on {gh_repo.full_name} -"+
+                        f"this is unexpected, continuing with first one of: {[(x.state, x.html_url) for x in prs]}")
 
     if prs.totalCount == 1:
-        pr = prs[0]
-        logging.info(f"Found PR #{pr.number} for branch {branch_name} on {gh_repo.full_name}: {pr.html_url}")
-
-    if pr.state == 'closed':
-        logging.warning(f"PR #{pr.number} is not open - new PR will be created")
-        if pr.is_merged():
-            logging.warning(f"PR #{pr.number} for '{branch_name}' branch is already merged but rebase.sh produced results")
-            _extra_msgs.append(f"PR #{pr.number} for '{branch_name}' was already merged but rebase.sh produced results")
+        pull_req = prs[0]
+        logging.info(f"Found PR #{pull_req.number} for branch {branch_name} on {gh_repo.full_name}: {pull_req.html_url}")
+
+    if pull_req.state == 'closed':
+        logging.warning(f"PR #{pull_req.number} is not open - new PR will be created")
+        if pull_req.is_merged():
+            logging.warning(f"PR #{pull_req.number} for '{branch_name}' branch is already merged but rebase.sh produced results")
+            _extra_msgs.append(f"PR #{pull_req.number} for '{branch_name}' was already merged but rebase.sh produced results")
         else:
-            _extra_msgs.append(f"PR #{pr.number} for '{branch_name}' exists already but was closed")
+            _extra_msgs.append(f"PR #{pull_req.number} for '{branch_name}' exists already but was closed")
         return None
-    return pr
+    return pull_req
 
 
 def generate_pr_description(branch_name, amd_tag, arm_tag, prow_job_url, rebase_script_succeded):
```

Comment:
This function is complaining with : `W0613: Unused argument 'branch_name' (unused-argument)`, I believe `branch_name` is not required here and can be removed, unless we missed anything.